packages/search-engine/src/components/resultsPage/results.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from "react";
+
+import Header from "./Header"
+import Map from "./Map"
+import Footer from "./Footer"
+import ListResults from "./ListResults"
+import Pagination from "./Pagination"
+import "./styles/styles.css"
+
+export const ResultsPage = () => {
+  
+  const [allPlaces, setAllPlaces] = useState([]);
+  
+  const GetDataFromApi = () => {
+    fetch('https://jsonplaceholder.typicode.com/todos')
```

Comment:
@Cristhian-Medina we shouldn't hardcode the URLs like this, because it is a bad practice, so what we can do is to read this information from the environment variables. What do you think?

Reply:
I also think it’s the right way, now, how and where do I define environment variables?